### PR TITLE
Update desktop-cross-report-drill-through.md

### PR DESCRIPTION
**Pull request by AldrinL**
for any report opened from sharedwithme, the cross-dirll through would not work, no matter it's shared from my workspace or app workspace. The only way is to add this user to the workspace and open the report from that workspace.

### CODE REVIEW

**Comment by PRMerger12** [comment]
@AldrinL : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/create-reports/desktop-cross-report-drill-through.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The drillthrough action opens the *target page* in the *target report*.
 This article shows you how to set up and use cross-report drillthrough for Power BI reports.
 
 > [!NOTE]
-> You can't use cross-report drillthrough with individually-shared [Shared with me reports](../collaborate-share/service-share-dashboards.md#share-a-dashboard-or-report) within **My Workspace**. To use cross-report drillthrough, you must access reports in the workspace from which they were shared.
+> You can't use cross-report drillthrough with individually-shared [Shared with me reports](../collaborate-share/service-share-dashboards.md#share-a-dashboard-or-report). To use cross-report drillthrough, you must access reports in the workspace from which they were a member of.
```
Comment (maggiesMSFT) [inline powerbi-docs/create-reports/desktop-cross-report-drill-through.md:30]:
```suggestion
> You can't use cross-report drillthrough with individually-shared [Shared with me reports](../collaborate-share/service-share-dashboards.md#share-a-dashboard-or-report). To use cross-report drillthrough, you must access reports in workspaces that you’re a member of.
```

**Comment by PRMerger9** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by maggiesMSFT** [comment]
Hi, @AldrinL -- Thanks for the edit! Great clarification. I tweaked the language a bit.

**Comment by maggiesMSFT** [comment]
#sign-off

**Pull request merged**